openapi-cli/src/test/resources/generators/client/ballerina/path_parameter_with_special_name.bal

```diff
@@ -0,0 +1,33 @@
+import ballerina/http;
```

Comment:
Licence header :)

Reply:
These are sample files used in test cases. Adding license header is a separate command line options and there are set of samples to check the license header generation.